src/main/resources/templates/modal/importModal.html

```diff
@@ -13,15 +13,12 @@ <h5 class="modal-title" id="import-modal" th:utext="#{screen.message.import.moda
                 Import
             </button>
         </div>
+        <p></p>
```

Comment:
Instead of using an empty <p> tag, I think we should add margin-top CSS to the the div below. You can quickly achieve that using the mt-1 Bootstrap class. Bootstrap has 5 levels for their margin and padding classes, so play with the levels to see what is appropriate.

Reply:
Just changed the instructions div class to include mt-2, pushed to origin